charts/nginz/values.yaml

```diff
@@ -53,7 +53,7 @@ nginx_conf:
   - /search/top
   - /search/common
 
-  default_client_max_body_size: "256k"
+  default_client_max_body_size: "512k"
```

Comment:
A comment how this "magic number" was calculated would be great (though not mandatory.) If I remember this correctly, there was some specific request for a specific amount of group members? :thinking: